src/pages/user-descriptor/line-graph.jsx

```diff
@@ -1,59 +1,45 @@
 import React from "react";
 import ReactEcharts from "echarts-for-react";
 
-// Expected Object
-function BossObject(Unit, Weight, Latitude, Longitude, Time, Gender) {
-  this.unit = Unit;
-  this.weight = Weight;
-  this.latitude = Latitude;
-  this.longitude = Longitude;
-  this.time = Time;
-  this.gender = Gender;
+// GraphQL
+import { gql } from "apollo-boost";
+import { useQuery } from "@apollo/react-hooks";
+
+const lgData = gql`
+  {
+    getPaginatedDescriptors(unit: "lb", start: 0, count: 400) {
+      value
+      unixTimestamp
+    }
+  }
+`;
+
+function DataFunction() {
```

Comment:
We need to follow coding conventions, if we keep to same style, this helps reviewers or others that come later read the code more easily. Function names should not start with captilization; use camelCase for everything. DataFunction -> dataFunction. Also, we know this is a function already, rather name this to something like descriptorsQuery.